otherlibs/threads/stdlib.ml

```diff
@@ -39,6 +39,20 @@ let invalid_arg s = raise(Invalid_argument s)
 
 exception Exit
 
+type _raw_backtrace
+external _get_raw_backtrace:
+  unit -> _raw_backtrace = "caml_get_exception_raw_backtrace"
+external _raise_with_backtrace: exn -> _raw_backtrace -> 'a
+  = "%raise_with_backtrace"
```

Comment:
This is nitpicking, but: I am guessing the `_` is suppose to mean that these identifiers are "for internal use"; however this is already expressed by not exposing these values in the interface. Moreover, underscores in OCaml have another meaning (disabling warning 32), so I would rather remove them.